acceptance/acceptance_test.go

```diff
@@ -57,31 +127,11 @@ func testAcceptance(t *testing.T, when spec.G, it spec.S) {
 	})
 }
 
-func lifecycleCmd(t *testing.T, name string, args ...string) *exec.Cmd {
-	cmdArgs := append(
-		[]string{
-			"run",
-			"-mod=vendor",
-			"-ldflags",
-			"-X github.com/buildpacks/lifecycle/cmd.Version=some-version " +
-				"-X github.com/buildpacks/lifecycle/cmd.SCMCommit=asdf123",
-			"./cmd/" + name + "/main.go",
-		}, args...,
-	)
-
-	wd, err := os.Getwd()
-	h.AssertNil(t, err)
-
-	cmd := exec.Command(
-		"go",
-		cmdArgs...,
-	)
-	cmd.Dir = filepath.Dir(wd)
-
-	return cmd
-}
-
-func parseCommand(command string) (binary string, args []string) {
-	parts := strings.Split(command, " ")
-	return parts[0], parts[1:]
+func buildBinaries(t *testing.T) {
+	cmd := exec.Command("make", "build-"+runtime.GOOS)
+	cmd.Dir = ".."
+	cmd.Env = append(os.Environ(), "BUILD_DIR="+dir, "LIFECYCLE_VERSION=some-version", "SCM_COMMIT=asdf123")
```

Comment:
A little pedantic but could we simply pass in `dir` here instead of having a "global" variable.

Reply:
`dir` is also used by the tests themselves https://github.com/buildpacks/lifecycle/blob/one-binary/acceptance/acceptance_test.go#L117